js/bulkhead-callback.js

```diff
@@ -279,16 +283,77 @@ var bulkheadCallBack = (function() {
         return match;
     };
 
+    var __validateEditorContent_BulkheadStep = function(content) {
+        var match = false;
+        try {
+            var pattern = "return null;\\s*}\\s*\\)\\s*;\\s*}\\s*" +
+            "@Bulkhead\\(50\\)\\s*public\\s*Service\\s*serviceForVFA";
+            var regExpToMatch = new RegExp(pattern, "g");
+            content.match(regExpToMatch)[0];
+            match = true;
+        } catch (ex) {
+
+        }
+        return match;
+    };
+
+    var __validateEditorContent_AsyncBulkheadStep = function(content) {
+        var match = false;
+        try {
+            var pattern1 = ";\\s*}\\s*" +
+                "@Asynchronous\\s*@Bulkhead\\s*\\(\\s*value\\s*=\\s*50\\s*,\\s*" + 
+                "waitingTaskQueue\\s*=\\s*50\\s*\\)\\s*" +
+                "public\\s*Future<Service>\\s*serviceForVFA";
```

Comment:
The line "public Future<Service> serviceForVFA" is actually the replacement of "public Service serviceForVFA". We need to add the readonly ending boundary "Service chatService =".

I'll slack Annie about why some lines are turned to readonly after replacement. Thank you.